examples/package.json

```diff
@@ -1,5 +1,5 @@
 {
   "dependencies": {
-    "pluggable-json": "github:juttle/pluggable-json"
+    "pluggable-json": "file:///Users/olegseletsky/git/juttle/pluggable-json"
```

Comment:
Ah oops, wanted to install from your local checked out code instead of npm, switched to using a relative path.